packages/paste-website/src/pages/components/toast/index.mdx

```diff
@@ -457,11 +475,16 @@ If you choose to handle state entirely yourself, you can use `<Toaster />` on it
 
 #### Toast Props
 
-| Prop       | Type         | Description                                                 | Default |
-| ---------- | ------------ | ----------------------------------------------------------- | ------- |
-| children   | `ReactNode`  |                                                             | null    |
-| onDismiss? | `() => void` | Create dismissable toast by providing an onDismiss callback | null    |
-| variant    | `string`     | `error` / `neutral` / `sucess` / `warning`                  | null    |
+| Prop             | Type         | Description                                                                               | Default |
+| ---------------- | ------------ | ----------------------------------------------------------------------------------------- | ------- |
+| children         | `ReactNode`  |                                                                                           | null    |
+| onDismiss?       | `() => void` | Create dismissable toast by providing an onDismiss callback                               | null    |
+| variant          | `string`     | `error` / `neutral` / `sucess` / `warning`                                                | null    |
+| i18nDismissLabel | `string`     | Label for the dismiss button in a dismissable toast. The default value is 'Dismiss toast' |
+| i18nErrorLabel   | `string`     | Icon label text for the `error` variant. The default value is '(error)'                   |
+| i18nNeutralLabel | `string`     | Icon label text for the `neutral` variant. The default value is '(information)'           |
+| i18nSuccessLabel | `string`     | Icon label text for the `success` variant. The default value is '(success)'               |
+| i18nWarningLabel | `string`     | Icon label text for the `warning` variant. The default value is '(warning)'               |
```

Comment:
Suggestion: Can we update this so the default field is populated for the i18n props? It'll be easier to scan that way.

Reply:
Done 🎉